packages/amplify-appsync-simulator/package.json

```diff
@@ -27,7 +27,7 @@
   },
   "dependencies": {
     "amplify-velocity-template": "1.4.5",
-    "aws-sdk": "^2.963.0",
+    "aws-sdk": "^2.1022.0",
```

Comment:
It looks like this is already up to 2.1024.0.

Reply:
oh I update, these changes are from Friday :D